### ISSUE TITLE
Could you expose onscroll?

### ISSUE THREAD

**Issue by rkalidhindi-tanium** [vasturiano/3d-force-graph#96]
Hi @vasturiano,

Shutting down nav controls while recentering makes sense since that is a set amount of time when movement is forced (I use cameraPosition with 3000 ms to get to the right place), but I decided that completely turning off nav controls during autorotate doesn't make sense. 
Instead I am trying to get the nav controls to turn off autorotate for a few seconds. Currently I am using onclick to turn off autorotate, but I thought it would be nice to let all three nav controls do that.  Would it be okay to expose onscroll? I'm not sure if right click is a different event or can somehow be accessed through onclick. 

Thank you!

**Comment by vasturiano** [comment]
@rkalidhindi-tanium you're probably looking for the [wheel event](https://developer.mozilla.org/en-US/docs/Web/API/WheelEvent), if you wish to detect zoom interactions. The good news is that you already have it, if you detect the event in the capture phase of your DOM element you can attach any functionality you wish, f.e.:
```
document.getElementById('mygraph').addEventListener('wheel', () => console.log('scroll detected'), true);
```

Likewise, you can capture the other events `click`, `mousemove`, etc.

**Comment by rkalidhindi-tanium** [comment]
I have been able to use `mousemove` and `click`, but I tried using `scroll` instead of `wheel`, so thank you for the heads up! 
Would it be possible to access `onmousedown`? `onclick` is mostly ok, but it doesn't trigger until you let go of the mouse. So if a user is trying to drag, the first drag won't work. Also it seems the right click only triggers `mousedown` on most browsers, not `onclick`.
